Tests/InfomaniakCoreTests/UTDecodeApiResponse.swift

```diff
@@ -0,0 +1,123 @@
+/*
+ Infomaniak Core - iOS
+ Copyright (C) 2023 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+@testable import InfomaniakCore
+import XCTest
+
+@available(iOS 13.0, *)
+final class UTDecodeApiResponse: XCTestCase {
```

Comment:
What do you think of using for each test, everytime `// GIVEN // WHEN //THEN` comments?

Here is it self evident what is tested, yet we may want to use this all the time everywhere.